pkg/api/status.go

```diff
@@ -159,3 +169,42 @@ func (s *Service) statusGetPeersHandler(w http.ResponseWriter, r *http.Request)
 	})
 	jsonhttp.OK(w, statusResponse{Snapshots: snapshots})
 }
+
+// statusGetHandler returns the current node status.
+func (s *Service) statusGetNeighborhoods(w http.ResponseWriter, _ *http.Request) {
+	logger := s.logger.WithName("get_status_neighborhoods").Build()
+
+	if s.beeMode == DevMode {
+		logger.Warning("status neighborhoods endpoint is disabled in dev mode")
+		jsonhttp.BadRequest(w, errUnsupportedDevNodeOperation)
+		return
+	}
+
+	neighborhoods := make([]statusNeighborhoodResponse, 0)
+
+	nhoods, err := s.statusService.NeighborhoodsSnapshot()
+	if err != nil {
+		logger.Debug("unable to get neighborhoods status", "error", err)
+		logger.Error(nil, "unable to get neighborhoods status")
+		jsonhttp.InternalServerError(w, "unable to get neighborhoods status")
+		return
+	}
+
+	if len(nhoods) == 0 {
+		jsonhttp.NotFound(w, "neighborhoods not found")
+		return
+	}
+
+	for _, n := range nhoods {
+		binaryAddr := ""
+		for _, b := range n.Address.Bytes() {
+			binaryAddr += fmt.Sprintf("%08b ", b)
```

Comment:
I think that the binary representation could a method of a potentially new type swarm.Neighborhood. Just a suggestion.

Reply:
Searched for such a thing but could not find it, `bitStr(` not the best naming